src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceExporterOptions.cs

```diff
@@ -49,8 +49,13 @@ public class DynatraceExporterOptions
         public string Prefix { get; set; }
 
         /// <summary>
-        /// Gets automatically added as metric labels/dimensions.
+        /// Gets automatically added as metric dimensions.
         /// </summary>
-        public IEnumerable<KeyValuePair<string, string>> Tags { get; set; }
+        public IEnumerable<KeyValuePair<string, string>> DefaultDimensions { get; set; }
+
+        /// <summary>
+        /// Automatically adds monitored entity metadata to metric labels (e.g. 'dt.entity.process_group_instance=PROCESS_GROUP_INSTANCE-46F1121843B79F56')
```

Comment:
```suggestion
        /// Automatically adds host and process metadata retrieved from the OneAgent as metric labels
```
The concrete keys are an implementation detail and subject to change, I wouldn't mention them here.